reco_utils/dataset/criteo_dac.py

```diff
@@ -110,34 +117,64 @@ def _get_schema(include_label=True):
   return schema
 
 
-def _load_datafile(local_cache_path="dac.tar.gz", dbutils=None):
+def criteo_urls(size):
+  """Return the url of criteo based on the input size
+
+  Args:
+    size (str): Size of criteo dataset. It can be "full" or "sample".
+
+  Returns:
+    str: url of Criteo dataset.
+
+  """
+  url = {
+    "full": CRITEO_URL_FULL,
+    "sample": CRITEO_URL_SAMPLE
+  }
+  return url[size]
+
+
+def download_criteo(size="full", filename="dac.tar.gz", work_directory="."):
+    """Download criteo dataset as a compressed file.
+
+    Args:
+        size (str): Size of criteo dataset. It can be "full" or "sample".
+        filename (str): Filename.
+        work_directory (str): Working directory.
+
+    Returns:
+        str: Path of the downloaded file.
+
+    """
+    url = criteo_urls(size)
+    return maybe_download(url, filename, work_directory)
+
+
+def _load_datafile(size, local_cache_path="dac.tar.gz", dbutils=None):
     """ Download and extract file """
 
     path, filename = os.path.split(os.path.realpath(local_cache_path))
 
     # Make sure the temporal tar file gets cleaned up no matter what
-    atexit.register(_clean_up, local_cache_path)
+    atexit.register(remove_filepath, local_cache_path)
```

Comment:
I'm thinking that we should rethink of having this optional, in both criteo and movielens. Right now we are using maybe_download, but we are always erasing the data

Reply:
good point. can we utilize tmp folder so that even the user forget to remove later, os automatically cleans up?